CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing Guidelines
+
+The KUDO Operators project accepts contributions via GitHub pull requests. This document outlines the process to help get your contribution accepted.
+
+## Sign the CLA
+
+Kubernetes projects require that you sign a Contributor License Agreement (CLA) before we can accept your pull requests.
+
+Please see https://git.k8s.io/community/CLA.md for more info.
+
+## How to Contribute to an Existing Operator
+
+1. Submit an issue describing your proposed change to the operator in question.
+2. The operator maintainer or repo owners will respond  to your issue promptly
+3. If your proposed change is accepted, and you haven't already done so, sign a Contributor License Agreement (see details above).
+4. Fork the desired repo, develop and test your code changes.
+5. Submit a pull request.
+
+### Requirements for the new Operators
+
+- Must include a `README.md`
+- Must include source GitHub repositories for images used in the Operator
+- Must pass the `kubectl kudo package verify` 
+- Must specify the minimum `KUDO` version and `Kubernetes` version in `operator.yaml`
+- Must provide tests for the Operator
+- Must follow the structure described in [getting started guide](https://kudo.dev/docs/developing-operators.html#getting-started)
+- Each file in `templates` directory must be of one Kubernetes object type
+- Should follow Kubernetes best practices
+- Should support Operator upgrades
```

Comment:
do we really require this? what if it's something simple that does not need a special plan for upgrades?

Reply:
I wanted to separate `must` and `should` and this I included in `should`.